src/main/java/org/opensearch/flowframework/workflow/ProcessNode.java

```diff
@@ -130,7 +131,12 @@ public CompletableFuture<WorkflowData> execute() {
                     return;
                 }
             }
-            CompletableFuture<WorkflowData> stepFuture = this.workflowStep.execute(input);
+            CompletableFuture<WorkflowData> stepFuture = null;
+            try {
+                stepFuture = this.workflowStep.execute(input);
+            } catch (IOException e) {
+                throw new RuntimeException(e);
+            }
```

Comment:
This section is entirely rewritten in #66, you may want to leave this class alone in your PR.

Reply:
Ignoring this file for review based on the above comment.

Reply:
rebased with latest change